### ISSUE TITLE
Field validation always required

### ISSUE THREAD

**Issue by omargobran** [braintree/android-card-form#86]
### General information

* SDK/Library version:4.2.0
* Android Version and Device:Sunmi P2 android pos device with Android 7.1.1 

### Issue description
There is an issue if an Edittext is needed to be shown on the screen but without blocking validation (i.e. the field should not be required in validation but only should be shown as a field) left to the user to decide to enter the required data or leave it empty.

**Comment by omargobran** [comment]
I have cloned the project and solved this issue on my own local branch. If you would like I could push it and you can check the changes.

**Comment by scannillo** [comment]
Hi @omargobran, you are welcome to open a PR!

Depending on the fields you are requesting to remove validation from, we may or may not be able to accept the changes. Please either open a PR or provide more specifics to the fields you are looking to allow to be optional.

**Comment by tolulekes** [comment]
Hi @omargobran has this issue been corrected in the code?

**Comment by omargobran** [comment]
No, it hasn't been solved yet, but I will be opening a PR asap.

**Comment by omargobran** [comment]
No @sarahkoop. Unfortunately, the code was on my old device and it got formatted. Has this issue not been solved yet? 
If so, I can rewrite my code and submit a PR for review.

**Comment by sarahkoop** [comment]
Hi @omargobran - The current expected behavior in `CardForm` is to allow a developer to configure which fields are required, but to validate any required fields (with the exception of `cardholderName`). If there are fields that you wold specifically like to display as optional, but not validate, we may be able to accept the change depending on the fields.
